src/templates/blog.js

```diff
@@ -29,6 +29,22 @@ const Blog = ({ data, pageContext }) => {
 
   const intl = useIntl()
 
+  const getStartedSectionData =
```

Comment:
add these data to /data/i18n/en.json and /data/i18n/zh.json, respectively, then use them with `FormattedMessage` to render data